tools/server-admin/src/main/resources/application.properties

```diff
@@ -161,4 +161,18 @@ quarkus.arc.ignored-split-packages=\
 
 quarkus.http.test-port=0
 
+# Disable a "lot of things" that are not needed for integration testing. Disabled in an effort to get hold
```

Comment:
mentions integration testing however the file seems to be a non-test resource?

Reply:
`%test` profile